src/index.html

```diff
@@ -16,17 +16,38 @@
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```

Comment:
HTML Formatting: The attributes of the `form` tag should be correctly formatted. Each attribute should start on a new line with 2-space indentation related to the tag.